Build/BuildConfig.json

```diff
@@ -0,0 +1,5 @@
+{
+    "BuildVersion": "21.6",
+    "BaselineVersion": "21.4.52563.54065",
+    "CurrentBuildVersionInMaster": "23"
```

Comment:
"Master" doesn't sound right here.

It's more like `MaxAllowedObsoleteVersion"? Or something in that sense?